homeassistant/components/zwave_me/const.py

```diff
@@ -0,0 +1,31 @@
+"""Constants for ZWaveMe."""
+# Base component constants
+NAME = "Z-Wave-Me"
+DOMAIN = "zwave_me"
+DOMAIN_DATA = f"{DOMAIN}_data"
+VERSION = "0.0.1"
+ATTRIBUTION = "Z-Wave-Me"
+ISSUE_URL = "https://github.com/Z-Wave-Me/ha-core/pulls"
+
+ZWAVEPLATFORMS = [
```

Comment:
Please separate words with underscore in variable names.
```suggestion
ZWAVE_PLATFORMS = [
```